search.html

```diff
@@ -3,36 +3,35 @@
     <head>
         <meta charset="UTF-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <title>Document</title>
+        <title>Searching</title>
         <link rel="stylesheet" href="src/search/css/style.css" />
+
+               <!-- Page icon -->
+               <link rel="icon" href="./src/assets/favicon.png">
 		<!-- style for the top more button -->
 		<link rel="stylesheet" href="src/home/css/morebutton.css" />
+
+		<!-- style for the top more button -->
+		<link rel="stylesheet" href="src/home/css/morebutton.css" />
+
   
         </style>
     </head>
     <body>
         <header>
             <div id="header-inner">
-                <a href="#" id="logo"></a>
-                 <a class="navbar-brand" rel="home" href="#" title="logo">
-                        <img src="src/assets/logo.png" width="90" height="80" >
-                <div class="container blue circleBehind">
-                    <a href="#" id="menu-icon"></a>
-                    <a href="index.html">Home</a>
-                    <a href="sort.html">Sorting</a>
-                    <a href="#" class="current">Searching</a>
-                    <a href="#">Path Finding</a>
-                    <a href="#">Graph</a>
-					<div class="dropdown">
-						<button onclick="show_hide()" class="Morebutton" > More </button>
```

Comment:
Same as previous comment.

Reply:
It was by mistake , I'll add it again , no issues.